org.civicrm.reqangular/src/common/decorators/ui-bootstrap/uib-modal.decorator.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env amd */
+
+define(function () {
+  'use strict';
+
+  return ['$delegate', '$document', function ($delegate, $document) {
+    var openFunction = $delegate.open;
+
+    $delegate.open = open;
+
+    /**
+     * Decorates uibModal to set prevent body from scrolling while a modal is open
+     * and revert to the original styles values when it is closed
+     *
+     * @param  {any}
+     * @return {Object} Modal Instance
+     */
+    function open () {
+      var modalInstance;
+      var lockScrollStyle = 'overflow: hidden;height: 100%;width: 100%;';
```

Comment:
Add a space after `;`, 
`'overflow: hidden; height: 100%; width: 100%;'`

Reply:
Why? This style is appended to the end of the current style.

Reply:
Sorry I meant space after `:`